test/unit_test/runtime/fmt_test.cpp

```diff
@@ -296,7 +296,7 @@ TEST_CASE("operator<< on a quantity", "[text][ostream][fmt]")
 
       SECTION("iostream")
       {
-        CHECK(stream.str() == "8 [1/100]m³");
+        CHECK(stream.str() == "8 [1 x 10⁻²]m³");
```

Comment:
We should use `×` Unicode sign here.

Reply:
done to match unicode literal from above